cpp/include/cuspatial/error.hpp

```diff
@@ -16,8 +16,9 @@
 
 #pragma once
 
-#include <cuda.h>
+//#include <cuda.h>
```

Comment:
We should remove this because it's redundant of `cuda_runtime_api.h`?

Reply:
Oops, I was testing commenting it out and forgot to remove it.  Yes, will remove.